docs/creature.md

```diff
@@ -55,6 +58,9 @@ Contains individual creature spawn data for each individual spawn of each indivi
 [21]: #npcflag
 [22]: #unitflags
 [23]: #dynamicflags
+[24]: #scriptname
```

Comment:
add description for scriptname

Reply:
no

Reply:
I'll also add the Comment column in this PR

Reply:
GH doesn't ping me on reviews fml